extensions/support-page/main.ts

```diff
@@ -6,7 +6,7 @@ export default class SupportPageMainExtension extends LensMainExtension {
       parentId: "help",
       label: "Support",
       click: () => {
-        this.navigate();
+        this.navigate("/support");
```

Comment:
Probably we don't even need to specify route? Default extension page url should be enough.

Reply:
No?